tests/classes/Utils.js

```diff
@@ -135,7 +135,7 @@ describe('Utils', () => {
 
       expect(() => {
         serverless.utils.readFileSync(tmpFilePath);
-      }).to.throw(new RegExp(`in "${tmpFilePath}"`));
+      }).to.throw(new RegExp('YAMLException:'));
```

Comment:
I think I found a balance with the regexp -- still checks for YAMLException and invalid.yml.

Or do you feel that the check for YAMLException makes the test too brittle?